### DISCUSSION TITLE
Simplifier la demande d'accès

### DISCUSSION THREAD

**Discussion by ghost** [betagouv/ma-cantine#1758] [Idées]
**Besoins :**
- Besoin formulé par une référente restauration en Région : pouvoir demander, en une seule fois, l'accès à toutes les cantines d'une même zone géographique et d'un même secteur. Ce serait utile pour les référents en Dpt ou Rég° qui n'ont pas créé les cantines et voudraient faire une demande d'accès groupée.
- Amélioration à laquelle j'ai pensé : plutôt que de devoir remplir le formulaire de création de cantine pour pouvoir en demander l'accès, avoir un bouton "Demander l'accès" directement sur la fiche d'une cantine dans "Nos Cantines".

Votre avis, @JenniferStephan, @hfroot, @alemangui ?

**Comment by alemangui** [comment]
À voir si ça n’entraîne des cas indésirables où l'on donnerait accès à des cantines qui n'appartient pas à la personne. Je pense par exemple aux groupes privés : si Sodexo met des cantines ou des cuisines centrales dans un département, ils n'ont peut être pas envie que toutes soient gérables par une personne ayant fait cette demande. Je comprends que c'est pas idéal de demander une par une, mais la conséquence en sécurité de donner un accès trop large peut être importante.

**Reply by ghost** [reply]
Je supposais que l'étape de la validation de l'accès – soit par l'un de nous si la cantine n'a pas de gestionnaires, soit par le ou l'un des gestionnaires existants si la cantine a déjà au moins un gestionnaire – permettait d'éviter justement de donner l'accès à des indésirables. Mais tu es plus compétent que moi en sécurité. Je vous laisse juger.
